### PR TITLE
Proposing LLM Instance Gateway

### PR DESCRIPTION
**Pull request by kfswain**
As discussed in: https://github.com/kubernetes/community/pull/8029

Wg-serving proposals will be made in this repo, this PR is the first of those, specifically proposing LLM Instance Gateway

### CODE REVIEW

File: proposals/012-llm-instance-gateway/README.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    - K8s Gateway API
+-  Other gateway features and projects, including high level LLM gateways  
+    - existing deployment tools like kserve or kaito
+    - different model servers
+
+### Non-Goals
+
+
+#### Proposal Non-Goals
+-   Creation of a fully realized KEP
```
Comment (smarterclayton) [inline proposals/012-llm-instance-gateway/README.md:65]: You can remove this here

**Comment by smarterclayton** [comment]
Two comments, then LGTM based on what we have discussed up until now.

File: proposals/012-llm-instance-gateway/README.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+As presented in the [demo](https://youtu.be/NUBZg_uqqXk?si=v681EeYdGUGEVqQQ&t=1458) and building further upon the [joint proposal](https://docs.google.com/document/d/1sFNHQqUWm1DIzC9GxXp3cKRm8cUtTcGuwZYkjkOkUqk/edit?tab=t.0#heading=h.9brozdsx9dqo), we are proposing that a gateway, focused on
+multiplexing 
+use cases upon shared hardware has distinct advantages in enabling efficient and fair use of multiple use-cases over a shared pool of compute.
+
+### Ownership
+As the joint proposal indictates, this effort is primarily being led by Google and Bytedance. 
+
+With *project owners* from each org: 
+- kfswain
+- Jeffwan
```
Comment (kfswain) [inline proposals/012-llm-instance-gateway/README.md:33]: @Jeffwan looking for your sign-off here! Thanks!

**Comment by Jeffwan** [comment]
One comment and rest part looks good to me.

File: proposals/012-llm-instance-gateway/README.md (hunk 2 above)
Comment (kfswain) [reply proposals/012-llm-instance-gateway/README.md]: Done

**Review by terrytangyuan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**